### PR TITLE
Improve google documentation

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: vdirsyncer/storage/google.py [hunk 1]
```diff
@@ -94,6 +94,11 @@ def _save_token(token):
 class GoogleCalendarStorage(dav.CaldavStorage):
     __doc__ = '''Google calendar.
 
+    .. note:: You have to poperly set up ``collections``, otherwise errors like
```
Comment (untitaker) [inline vdirsyncer/storage/google.py]: Not sure if that's useful. `collections` always needs to be properly set up?

Comment (untitaker) [inline vdirsyncer/storage/google.py]: At this point I'm actually curious which problem led you to write this. Stuff like #463? Though that typo should be properly caught when parsing the config...

Comment (evgeni) [inline vdirsyncer/storage/google.py]: The example in "tutorial" (for ownClould) has `collections = null` and "config" does not list that as a required field. Thus I thought one only has to set `collections` when syncing non-default adressbooks/calendars.

Comment (untitaker) [inline vdirsyncer/storage/google.py]: Ah, so I should fix the google storage to reject `collections = null`.

Comment (evgeni) [inline vdirsyncer/storage/google.py]: If that's feasible, yeah.
Should I drop the changes to `google.py` from that PR then?

Comment (untitaker) [inline vdirsyncer/storage/google.py]: I think https://github.com/pimutils/vdirsyncer/pull/466 would fix this, so you can take this paragraph out.

Comment (untitaker) [inline vdirsyncer/storage/google.py]: Sorry, didn't get a push notification about your reply... yes, please drop those changes.

**Comment by evgeni** [comment]
@untitaker done :)

**Comment by untitaker** [comment]
Thanks!

**Pull request merged**